blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/PipelineSearch.java

```diff
@@ -40,6 +44,7 @@
  */
 @Extension
 public class PipelineSearch extends OmniSearch<BluePipeline>{
+    private static final String STARTSWITH_PARAM="startsWith";
```

Comment:
@imeredith regex related code doesn't exists, thats what I am suggesting to be implemented instead of adding startsWith param.